server/pipeline_trigger.go

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package server
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strings"
+)
+
+func getPipelineFormData(pipelineTrigger *PipelineTrigger, args []string) url.Values {
+	data := url.Values{}
+	data.Add("token", pipelineTrigger.Token)
+	data.Add("ref", pipelineTrigger.Reference)
+	for key, value := range pipelineTrigger.Variables {
```

Comment:
nit: can we name these parameters something like `variableName` and `variableValue` so it's more obvious what we're comparing below, especially when we also then iterate over args?

Reply:
Replaced with `variableName` and `variableValue`

https://github.com/mattermost/matterbuild/pull/92/files#diff-3a2928403027668f297d9bce542b27fe04a47b86650a36900a1e408018da70e5R20-R29